setup.py

```diff
@@ -1,9 +1,17 @@
-from distutils.core import setup
+from setuptools import (
+    setup,
+    find_packages
+)
+from datetime import datetime
+
+version = "0.1.0+{}".format(datetime.now().strftime("%Y%m%d.%H%M%S"))
```

Comment:
What's the thinking here? I like not having to manually change the build, but I've never seen a python module do this.

I'd prefer the builds be repeatable, and having short version numbers (e.g. `0.1.1`) is much simpler in a `requirements.txt` file.